build.sbt

```diff
@@ -13,62 +15,73 @@ val commonSettings = Seq(
   .in(file("."))
   .disablePlugins(MimaPlugin)
   .settings(commonSettings)
   .settings(
-    name := "play-json-tools",
-    publish / skip := true,
+    publish / skip := true
   )
   .aggregate(
-    playJsonTools,
-    playJsonGeneric,
-    playJsonJsoniter
+    `play-json-tools`,
+    `play-json-genericJVM`,
+    `play-json-genericJS`,
+    `play-json-jsoniterJVM`,
+    `play-json-jsoniterJS`
   )
 
+lazy val `play-json-genericJVM` = `play-json-generic`.jvm
+
+lazy val `play-json-genericJS` = `play-json-generic`.js
+  .settings(
+    coverageEnabled := false // FIXME: No support for Scala.js 1.0 yet, see https://github.com/scoverage/scalac-scoverage-plugin/pull/287
+  )
 
-lazy val playJsonGeneric = project
-  .in(file("play-json-generic"))
+lazy val `play-json-generic` = crossProject(JVMPlatform, JSPlatform)
+  .crossType(CrossType.Pure)
   .settings(commonSettings)
-  .settings(Seq(
-    moduleName := "play-json-generic",
-    name       := "play-json-generic",
+  .settings(
     scalacOptsFailOnWarn := Some(false),
     libraryDependencies ++= Seq(
-      shapeless,
-      playJson,
-      scalaTest % Test,
-    ).map(excludeLog4j)))
-
+      "com.chuusai" %%% "shapeless" % "2.3.3",
+      "com.typesafe.play" %%% "play-json" % "2.9.2",
+      "org.scalatest" %%% "scalatest" % "3.2.7" % Test,
+    ).map(excludeLog4j)
+  )
 
-lazy val playJsonTools = project
-  .in(file("play-json-tools"))
+lazy val `play-json-tools` = project
   .settings(commonSettings)
-  .settings(Seq(
-    moduleName := "play-json-tools",
-    name       := "play-json-tools",
+  .settings(
     libraryDependencies ++= Seq(
-      playJson,
-      nel,
-      scalaTest % Test,
-    ).map(excludeLog4j)))
+      "com.typesafe.play" %% "play-json" % "2.9.2",
+      "com.evolutiongaming" %% "nel" % "1.3.4",
+      "org.scalatest" %% "scalatest" % "3.2.7" % Test,
+    ).map(excludeLog4j)
+  )
+
+lazy val `play-json-jsoniterJVM` = `play-json-jsoniter`.jvm
 
-//++ 2.12.10 or ++ 2.13.1
-lazy val playJsonJsoniter = project
-  .in(file("play-json-jsoniter"))
+lazy val `play-json-jsoniterJS` = `play-json-jsoniter`.js
+  .settings(
+    coverageEnabled := false // FIXME: No support for Scala.js 1.0 yet, see https://github.com/scoverage/scalac-scoverage-plugin/pull/287
+  )
+
+lazy val `play-json-jsoniter` = crossProject(JVMPlatform, JSPlatform)
+  .crossType(CrossType.Full)
   .settings(commonSettings)
-  .settings(Seq(
-    moduleName := "play-json-jsoniter",
-    name       := "play-json-jsoniter",
-    libraryDependencies ++= {
-      CrossVersion.partialVersion(scalaVersion.value) match {
-        case Some((2, v)) if v >= 13 =>
-          Seq(playJson, nel, jsoniter, scalaTest % Test, jsonGenerator % Test).map(excludeLog4j)
-        case _ =>
-          Seq(playJson, nel, jsoniter, collectionCompact, scalaTest % Test).map(excludeLog4j)
-      }
-    }))
+  .settings(
+    libraryDependencies ++= (Seq(
+      "com.typesafe.play" %%% "play-json" % "2.9.2",
```

Comment:
could you please move this out back to Dependencies files so we have single place with define versions?

Reply:
I reverted Dependencies.scala with updated versions of dependencies